linuxNew/README.md

```diff
@@ -8,7 +8,7 @@ and then looping over configuration to create the various packages and signing t
 with the (Eclipse Foundation as a default) signing service.
 
 TODO You can then optionally upload those packages to a package repository of your choice.
-TODO List the default location that Adoptium is using.
+The default Adoptium package repository is https://packages.adoptium.net/ui/packages.
 
```

Comment:
Perhaps add a sentence here linking to the jenkins job that we have to make it clear that the normal process is to use the job instead of running it manually.

Reply:
The job may not be visible to anyone. Updated with the sentence says the packages are built and uploaded by jenkins pipeline, which probably be good enough.